plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleDependencyResolutionHelper.java

```diff
@@ -401,6 +444,31 @@ private static void printDependencyMap(Map<DependencyDescriptor, Set<DependencyD
         return value;
     }
 
+    /*
+     * Debugging the logic within class is quite challenging since the execution happens within the Gradle daemon. The
+     * below method will launch a JFrame and display the message that you provide. Uncomment when debugging this class.
+     *
+     * @param message the message to display in the UI.
+     */
+    /*
+    private static void displayMessage(String message) {
```

Comment:
I hope you do not mind me adding some debug logic in its commented form. Let me know if you prefer not having this in the code.